Directory.Packages.props

```diff
@@ -3,6 +3,15 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
+    <!-- AWS SDK for .NET dependencies -->
```

Comment:
It would be great to get these packages added to the Aspire NuGet feeds. Some of these are newer versions of packages that are already in there. Currently I'm testing with hacking the `NuGet.config` to allow the public NuGet feed.

Reply:
Are these the versions you need?

Reply:
> Currently I'm testing with hacking the NuGet.config to allow the public NuGet feed.

Would it be possible to use the "hack" while the PR was in draft, and adding more dependencies? (and possibly updating new versions that are released). Then once the PR is ready for review, we can revert the nuget.config change, and mirror the necessary packages to the trusted feed.